Documentation/General/productVision.tex

```diff
@@ -0,0 +1,155 @@
+\documentclass[10pt,a4paper]{article}
+\usepackage[utf8]{inputenc}
+\usepackage{amsmath}
+\usepackage{amsfonts}
+\usepackage{amssymb}
+\usepackage{cite}
+\usepackage{hyperref}
+\usepackage{graphicx}
+
+
+\begin{document}
+
+\title{Product Vision\\
+	\begin{small}
+	The why, who and what
+	\end{small}
+}
+% Question: should this be our names or organization name?
+\author{MoodCat.me}
+\maketitle
+
+\section{Why should we build MoodCat?}
+Various competitors provide a system that can offer users popular songs or songs that are similar to songs the user already listened to.
```

Comment:
Various competitors may not be the best start for the entire product vision. Maybe use: Music listening services.